README.md

```diff
@@ -25,7 +25,11 @@ To build this project, please execute:
 mvn package -DskipTests
 ```
 
-`mvn package` will assemble all the required dependencies and package into an uber jar.
+`mvn package` will assemble all the required dependencies and package into an uber jar:
```

Comment:
Is this connector supposed to be always built from source, or are ready-made uber jar downloads available somewhere?